src/modules/categories/components/ContentMatcher.js

```diff
@@ -0,0 +1,48 @@
+// @flow
```

Comment:
ContentMatcher is no component. Therefore it should be moved one level up.

Reply:
We also usually don't have classes for functionality, do we? Of course this is also okay, just not sure if a class makes sense here.

Reply:
This could also not really category specific but could be used for events, ... Maybe it should even live somewhere else, idk.

Reply:
Yeah the function could just be exported as `export const ...`

Reply:
I think it was @Schedulaar who recommended using a separate class for the ContentMatcher in the webapp, as it simplifies testing. 



> Yeah the function could just be exported as `export const ...`

Do you mean that I scrap the class and keep the functions as local functions? My question then would be, whether it is possible to to test the functions the and whether I should apply those changes to the webapp.